node/node.go

```diff
@@ -112,7 +121,7 @@ func NewNode(ctx context.Context, opts ...NodeOpt) (*Node, error) {
 	if err != nil {
 		return nil, err
 	}
-	db, err := db.NewDB(ctx, rootstore, options.dbOpts...)
+	db, err := db.NewDB(ctx, rootstore, acp.NoACP, options.dbOpts...)
```

Comment:
question: Why not also dependent on the default options like for `rootstore` above?